apps/sms/js/subject_composer.js

```diff
@@ -106,7 +113,8 @@
 
       // methods
       updateValue: updateValue.bind(this),
-      updatePlaceholder: updatePlaceholder.bind(this)
+      updatePlaceholder: updatePlaceholder.bind(this),
+      updateDirection: updateDirection.bind(this)
```

Comment:
hmm you are right... maybe we can revisit if it's necessary to be that "private" for communication inide the component.